### ISSUE TITLE
[FEATURE] Improve dark mode

### ISSUE THREAD

**Issue by MHillier98** [compodoc/compodoc#1092]
##### **Overview of the feature**

Improving the default theme colours of the dark mode would make it easier on the eyes.

In my opinion, it would be nice to follow the Angular docs dark theming, which can also be toggled between light and dark modes. https://angular.io/guide/accessibility

The main issue IMO is the red font used for code blocks, which makes it hard to read against the black background. For example:
![image](https://user-images.githubusercontent.com/26564949/125061497-bf3ab280-e0f0-11eb-8afa-d6c9850cee53.png)

The blue links are much easier to read: 
![image](https://user-images.githubusercontent.com/26564949/125061644-e1cccb80-e0f0-11eb-9318-7d6f0c1cb24a.png)

##### **Motivation for or Use Case**

While the new dark mode is a great first step, I'm currently using Compodoc in a different browser so I don't have to use dark mode for it. As I'm using dark mode for literally every other app, having a usable dark mode would be nice for me.

##### **Related issues**

Previous discussion in https://github.com/compodoc/compodoc/issues/1064 and implemented in https://github.com/compodoc/compodoc/commit/2bb51cb29c00402e925cc94c16dafcf5e494a237.

**Comment by MHillier98** [comment]
Thanks for the quick fix! Looks way better now!

A few other things:

1. The hover menu's hover makes a white block:
![image](https://user-images.githubusercontent.com/26564949/125093886-2028b180-e116-11eb-89cc-ca944cd4d13d.png)

2. The parameters table header row has black text:
![image](https://user-images.githubusercontent.com/26564949/125093999-3e8ead00-e116-11eb-83c6-20f9b76a69ce.png)

3. Tabs at the top of pages aren't visible unless selected:
![image](https://user-images.githubusercontent.com/26564949/125094381-8dd4dd80-e116-11eb-833f-84e8aadba12b.png)

Looks good to me other than these 3 things :)

**Comment by vogloblinsky** [comment]
@MHillier98 Are you sure you are up-to-date or cache cleared in your web browser, i cannot reproduce your last remarks.

**Comment by MHillier98** [comment]
@vogloblinsky Yes I am on 1.1.13 and have cleared my cache. I think it's because I'm using the material theme.
